sklearn/ensemble/tests/test_iforest.py

```diff
@@ -295,6 +295,34 @@ def test_score_samples():
                        clf2.score_samples([[2., 2.]]))
 
 
+@pytest.mark.filterwarnings('ignore:default contamination')
+@pytest.mark.filterwarnings('ignore:behaviour="old"')
```

Comment:
why is this needed here?

Reply:
yes it is, sorry for the noise.

Reply:
no problem at all, thanks for your review